### PR TITLE
[fix] doStream exception for initial chunk arguments

### PR DESCRIPTION
**Pull request by minpeter**
This PR addresses an issue where doStream ignores function arguments when a provider sends them in the first chunk of the streamed response. The current implementation assumes that the first chunk will not contain function arguments, leading to the omission of these arguments and the subsequent failure of the doStream method.

### Changes:

- **Test Update**: Added a test case in openai-chat-language-model.test.ts to simulate a scenario where arguments are included in the first chunk of a streamed response.
- **Code Update**: Modified openai-chat-language-model.ts to handle cases where function arguments are present in the initial chunk. The function now checks for non-empty arguments and processes them as tool-call-delta events.

### Reason:  
When using the streamObject() function in tool mode with FriendliAI serverless, the first chunk's arguments were being ignored, causing incomplete data processing. This was due to a lack of explicit documentation in the OpenAI API, necessitating a fix to ensure robust handling of all streamed chunks, including those that contain arguments from the start.

This fix ensures that the doStream function correctly processes all parts of the response, preventing errors and ensuring that the full payload is accurately streamed and handled.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (seuha516) [inline pnpm-lock.yaml:1]: unnecessary change?

File: packages/openai/src/openai-chat-language-model.ts [hunk 1]
```diff
@@ -476,6 +476,16 @@ export class OpenAIChatLanguageModel implements LanguageModelV1 {
                       toolName: toolCall.function.name,
                       args: toolCall.function.arguments,
                     });
+
+                    // Some providers send arguments starting from the first chunk
```
Comment (seuha516) [inline packages/openai/src/openai-chat-language-model.ts:480]: 주석 indent 2칸 앞으로?

File: packages/openai/src/openai-chat-language-model.test.ts
Comment (seuha516) [inline packages/openai/src/openai-chat-language-model.test.ts:1]: Is all other tests passed?

**Review by seuha516 — APPROVED** [review]
Otherwise LGTM

File: packages/openai/src/openai-chat-language-model.ts (hunk 1 above)
Comment (minpeter) [reply packages/openai/src/openai-chat-language-model.ts:480]: lint에 의해 불가능

File: packages/openai/src/openai-chat-language-model.test.ts
Comment (minpeter) [reply packages/openai/src/openai-chat-language-model.test.ts:1]: Yes! And I also did manual tests to see if it works on openai and friendliai.

**Review by seuha516 — APPROVED** [review]
